Mastodon/Scene/Profile/MeProfileViewModel.swift

```diff
@@ -29,5 +29,27 @@ final class MeProfileViewModel: ProfileViewModel {
             }
             .store(in: &disposeBag)
     }
-    
+
+    override func viewDidLoad() {
+
+        super.viewDidLoad()
+
+        Task {
+            do {
+
+                _ = try await context.apiService.authenticatedUserInfo(authenticationBox: authContext.mastodonAuthenticationBox).value
+
+                try await context.managedObjectContext.performChanges {
+                    guard let me = self.authContext.mastodonAuthenticationBox.authentication.user(in: self.context.managedObjectContext) else {
+                        assertionFailure()
+                        return
+                    }
+
+                    self.me = me
+                }
+            } catch {
+                // do nothing?
```

Comment:
Hm, maybe we should throw some error at the user, but not sure what the reason for such errors might be and if they're resolvable. Maybe let's create an issue for this to resolve this later?